docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_polymake.md

```diff
@@ -0,0 +1,55 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_polymake.md"]
+```
+
+
+# `Polyhedron` and `polymake`'s `Polytope`
+
+Many polyhedral computations are done through `polymake`.
+This is visible in the structure `Polyhedron` via a pointer `pm_polytope` to the corresponding `polymake` object.
+This allows to apply all functionality of `polymake` by calling a suitable `Polymake.jl` function on this pointer.
```

Comment:
Again, the read may not know anything about `Polymake.jl` here. So perhaps also introduce that?